inst/include/Rcpp/sugar/functions/is_na.h

```diff
@@ -33,11 +33,11 @@ class IsNa : public ::Rcpp::VectorBase< LGLSXP, false, IsNa<RTYPE,NA,VEC_TYPE> >
 
 	IsNa( const BASE& obj_) : obj(obj_){}
 
-	inline int operator[]( int i ) const {
+        inline int operator[]( R_xlen_t i ) const {
```

Comment:
We are not terribly consistent.  Four spaces.  Doing without tabs seems better (as we learned over the years), but we are religious enough to filter all files.  

I use Emacs to adjust indentation when I open a file and see something wrong.